__std__/cells/plutus-apps/library/plutus-apps-project-with-haddock.nix

```diff
@@ -0,0 +1,6 @@
+{ inputs, cell }:
+
+cell.library.make-plutus-apps-project {
+  deferPluginErrors = true;
```

Comment:
Why do we want to `deferPluginErrors = false` for CI, but `deferPluginErrors = true` for generating haddocks?
Why not use `deferPluginErrors = true` for CI also?

Reply:
I think we want to know when plugin errors happen. For haddocks we turn it off because it doesn't work.